products/appengine/terraform.yaml

```diff
@@ -107,7 +107,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "web_service"
         vars:
           bucket_name: "appengine-test-bucket"
-
+  ServiceSplitTraffic: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "{{service_id}}"
```

Comment:
`id_format` and `import_format` are probably redundant with what's used by default. Did you find that you needed custom values?

Reply:
Resolving since this is really old.